Week-3/stack.js

```diff
@@ -0,0 +1,18 @@
+function createStack() {
+    let items = [];
+    return {
+        items: items,
+        push(item) {
+            items.push(item);
+        },
+        pop() {
+            items.pop();
+        }
+    }
+}
+
+const stack = createStack();
+stack.push(34);
+console.log(stack.items);
+
+// This question is not very clear to me
```

Comment:
Does this question still need clarification? If yes, lets catch up, else please remove this.